src/three/controls/EnvironmentControls.js

```diff
@@ -399,6 +400,17 @@ export class EnvironmentControls extends EventDispatcher {
 
 			}
 
+			// Add mouse click handle in pointerMove
+			if ( this.state === ZOOM ) {
+
+				const delta = e.movementY;
+				// use LOG to scale the scroll delta and hopefully normalize them across platforms
+				const deltaSign = Math.sign( delta );
+				const normalizedDelta = Math.log( Math.abs( delta ) + 1 );
+				this.zoomDelta -= 3 * deltaSign * normalizedDelta * 1;
```

Comment:
Log scale shouldn't be needed if we're moving in screen space. It's used because the reported wheel "scroll" has pretty severely different magnitudes depending on platform.